powerauth-restful-model/src/main/java/io/getlime/security/powerauth/rest/api/model/request/ActivationLayer1Request.java

```diff
@@ -17,7 +17,7 @@
  * You should have received a copy of the GNU Affero General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-package io.getlime.security.powerauth.rest.api.model.request.v3;
+package io.getlime.security.powerauth.rest.api.model.request;
```

Comment:
🤔 Do we want to keep the package for the version (for example when we introduce `v4`)?

Reply:
Yes, there are some minor differences between `v3` and future `v4` model. I would keep `v3` for now and we'll see whether it's efficient to cover v3 as a subset of v4 API or we'll do the full separation.

Reply:
Ok, I have reverted the commit with `v3` removal.

Reply:
Once again, let's drop the `v3` package.